angular-app/src/app/workspace/workspace.component.ts

```diff
@@ -78,6 +78,14 @@ export class WorkspaceComponent implements OnInit {
       });
     }
 
+    randomColor(): String {
```

Comment:
While interesting, I do think this function has the potential to generate some not-so-great colors --- for example, something that doesn't contrast well with the background and makes the waveform difficult to see.

Reply:
Yeah, after thinking more about it, I think it should be a cycle between three colors. Red, blue, and green for example.

Reply:
Cool!

Reply:
Is the reason for changing the color to something else a visual cue to the user that their audio file was successfully loaded? If so, I would recommend a temporary display of some text indicating that the <name of audio file> was successfully uploaded or displayed.

Changing colors may indicate to the user that the different colors have some significance and be confused as to why this file is blue and not red or green, for example.

Reply:
I figured that a single color would be a bit "dull" to see over and over again. But the color of the waveform doesn't have to change necessarily. I can see how it could confuse a user as to why they're seeing different colors.